### PR TITLE
Cloud 218 redis roaring fork

### PR DESCRIPTION
**Pull request by tvaughanRR**
Updating GHA for our ECR

### CODE REVIEW

File: .github/actions/build-and-push-docker-image/action.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Build, tag, and push image to Amazon ECR
+      id: build-image
+      env:
+        AWS_ECR: ${{ inputs.aws_ecr }}
+      run: |
+        docker buildx build -t $AWS_ECR/redis-roaring:sha_$GITHUB_SHA --platform=linux/arm64 --output type=docker .
```
Comment (dsteadman) [inline .github/actions/build-and-push-docker-image/action.yml]: What environments are we building this for? Our AWS resources run Intel, but I get if we want to also build AMD for use in local envs

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -1,67 +1,77 @@
...
-    - name: Test
-      run: ./performance.sh
-  docker:
-    name: Build and push to Dockerhub
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Configure
+        run: ./configure.sh
+      - name: Install valgrind
+        run: sudo apt-get install valgrind
```
Comment (dsteadman) [inline .github/workflows/ci.yml:58]: nice! should we use this elsewhere?

File: .github/actions/build-and-push-docker-image/action.yml (hunk 1 above)
Comment (tvaughanRR) [reply .github/actions/build-and-push-docker-image/action.yml]: Oh; I was going off the spacey example... 

Assuming we want our ECR image to be the thing we deploy to our AWS accounts, it sounds like I should get rid of the `--platform` flag?

File: .github/workflows/ci.yml (hunk 2 above)
Comment (tvaughanRR) [reply .github/workflows/ci.yml:58]: Perhaps?  It's coming directly out of the underlying repo I forked this off of so I kept it

File: .github/actions/build-and-push-docker-image/action.yml (hunk 1 above)
Comment (dsteadman) [reply .github/actions/build-and-push-docker-image/action.yml]: Yep, you might not even need buildx either

Comment (tvaughanRR) [reply .github/actions/build-and-push-docker-image/action.yml]: Removed buildx and that seems to still work fine.  Good suggestion!

**Review by dsteadman — APPROVED** [review]

**Pull request merged**